pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -140,7 +140,7 @@ func Test_SubscribeLeafStream(t *testing.T) {
 	}()
 
 	//FIXME Waiting for subscribe to finish properly --> when event is issued assuring state consistency we can remove
-	time.Sleep(100000)
+	time.Sleep(15 * time.Second)
```

Comment:
I think 15 seconds is a bit excessive - because this means every time the test is run it will stop here for 15 seconds

in my own pull request I changes to 100*time.Millisecond and it worked

The original value above is in nano seconds, so 100ms is a 1000 fold increase